packages/blade/src/components/Icons/iconMap.ts

```diff
@@ -74,4 +74,6 @@ const iconMap: Record<string, IconComponent> = {
   TrendingDownIcon: TrendingDownIconComponent,
 };
 
+export const iconMapKeys = Object.keys(iconMap);
```

Comment:
Unfortunately this will still effect the bundle. :( 
Since we are importing all the icons here. 

Possible solution would be to maintain an array of all the icon names as strings only.

Reply:
Right, that would be hard to keep in sync though.